### PR TITLE
hpc: Add test for tool allowing to migrate from SLES to SLE HPC

### PR DESCRIPTION
**Pull request by asmorodskyi**
prove run http://kimball.arch.suse.de/tests/1547

### CODE REVIEW

**Comment by asmorodskyi** [comment]
@czerw I decided to split it into the separate commit. After merging this one I will just erease same file from  #5650

File: tests/hpc/migration.pm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    my $scc_code = get_required_var('SCC_REGCODE_HPC');
+    select_virtio_console();
+
+    assert_script_run('ls /etc/products.d/baseproduct');
+    zypper_call('in switch_sles_sle-hpc');
+    script_run('SUSEConnect -s');
+    assert_script_run('SUSEConnect -p sle-module-hpc/12/x86_64');
+    assert_script_run('SUSEConnect -p sle-module-web-scripting/12/x86_64');
+    assert_script_run("switch_to_sle-hpc -e testing\@suse.com -r $scc_code");
+    assert_script_run('ls /etc/products.d/baseproduct');
```
Comment (dzedro) [inline tests/hpc/migration.pm]: Didn't you want to do `/etc/products.d/` ?

**Comment by asmorodskyi** [comment]
@dzedro fixed . Do you think I need another prove run for this change ?

**Comment by dzedro** [comment]
I like more `ls -lA`, you don;t need new run

**Pull request merged**